.github/workflows/python-package.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     runs-on: ubuntu-latest  # It switched to 22.04 shortly after 2022-Nov-8
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9, "3.10", "3.11", "3.12-dev"]
+        python-version: [3.7, 3.8, 3.9, "3.10", "3.11", "3.12"]
```

Comment:
Do you know whether we can also add "3.13-dev" here?

Reply:
No, you can't yet.

Reply:
OK then.